### ISSUE TITLE
Styled markdown regions show as diff changes

### ISSUE THREAD

**Issue by nrichards** [jduckles/pwdiff#4]
Environment: macOS Sonoma 14.2.1, pandoc 3.1.9, git version 2.35.1

1. Add this line to both OLD and NEW:

```
# Tests
 
Testing whether *single asterisk* shows differently.

Testing whether **double asterisk** shows differently.

Testing whether ***triple asterisk*** shows differently.

Testing whether _single underscore_ shows differently.

Testing whether __double underscore__ shows differently.

Testing whether ~single tilde~ shows differently.

Testing whether ~~double tilde~~ shows differently.
```

2. Generate

3. Result shows the `**bold**` as a green, new change, and others:

<img width="404" alt="image" src="https://github.com/jduckles/pwdiff/assets/704698/08e83910-5434-4605-ba1f-8c6253f538bb">

Expected:

No green, new change indicator for `**bold**` and others.

**Comment by nrichards** [comment]
A solution could be to take advantage of how HTML comments are passed-through, e.g. `<!-- removed -->` and `<!-- added -->`.

Currently the tool uses `[-` `-]` `{+` and `+}` to mark removed and added regions.

**Comment by jduckles** [comment]
Fixed by merging #5
